### PR TITLE
add graphql on kyma blog post

### PR DESCRIPTION
**Pull request by hansamann**
This includes a guest author blog post for the kyma blog. It's based on a previous discussion with Lukasz Gornicki. I agree to all guidelines and sold my soul. Please have a read, I hope we can publish this soon on the kyma blog.

### CODE REVIEW

File: content/blog-posts/2019-10-24-graphql-on-kyma/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- [graphql-console-ui](https://github.com/hansamann/synolo/blob/master/graphql-console-ui/README.md): This project includes a minimal and basic Vue-based UI for the Kyma console. It includes the Kyma resources necessary to add an item to the Kyma console and includes the basic UI for editing the list of available types and resolvers via simple HTML text areas. This UI communicates with the graphql-api.
+- [graphql-console-api](https://github.com/hansamann/synolo/blob/master/graphql-console-api/README.md): The graphql-api uses the Kubernetes node.js API to load, edit and update the ConfigMaps for types and resolvers. It also enables the graphql-ui to restart the GraphqQL server, which essentially kills all pods of a graphql deployment for reloading them with the latest ConfigMaps.
```
[the remark this answers is not in the document]
Comment (hansamann) [reply content/blog-posts/2019-10-24-graphql-on-kyma/index.md]: Not sure if I understand the question 100% - but the configmaps are essentially text files and the pods get the content loaded at the point the pods are created. If you change a configmap while the pod is running, the changes are not immediately visible to the pod and it has no way to figure out the underlying config has changed. A quick and effective way to make the pods honor the new data/config is to "kill" them, Kubernetes will then recreate them with the current data/config.

File: content/blog-posts/2019-10-24-graphql-on-kyma/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- [graphql-console-ui](https://github.com/hansamann/synolo/blob/master/graphql-console-ui/README.md) - this project includes a minimal and basic Vue-based UI for the Kyma console. It includes the Kyma resources necessary to add an item to the Kyma console and includes the basic UI for editing the list of available types and resolvers via simple HTML text areas. This UI communicates with the graphql-api.
+- [graphql-console-api](https://github.com/hansamann/synolo/blob/master/graphql-console-api/README.md) - the graphql-api uses the Kubernetes node.js API to load, edit and update the ConfigMaps for types and resolvers. It also enables the graphql-ui to restart the GraphqQL server, which essentially kills all pods of a graphql deployment for reloading them with the latest ConfigMaps.
```
Comment (majakurcius) [inline content/blog-posts/2019-10-24-graphql-on-kyma/index.md]:
```suggestion
- [graphql-console-api](https://github.com/hansamann/synolo/blob/master/graphql-console-api/README.md) - the graphql-api uses the Kubernetes node.js API to load, edit and update the ConfigMaps for types and resolvers. It also enables the graphql-ui to restart the GraphqQL server, which essentially kills all pods of a graphql deployment to recreate them with the latest ConfigMaps.
```

**Comment by hansamann** [comment]
Thx for all the feedback and improvements. Please let me know what are the next steps. Of course I'd like to see this live pretty soon.

**Pull request merged**